src/js/stores/transforms/AppFormFieldToModelTransforms.js

```diff
@@ -147,6 +147,14 @@ const AppFormFieldToModelTransforms = {
         }
         delete definition.consecutiveKey;
         delete definition.isRandomPort;
+        if (definition.vip != null) {
```

Comment:
Is there a reason why you first store the vip in `definition.vip` and then trasnform it to `definition.labels["VIP_0"]`?